region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -1944,8 +1972,47 @@
       "link": [4, 4],
       "name": "Zoa Farm",
       "requires": [
-        {"refill": ["Energy", "Missile", "Super"]}
-      ]
+        {"or": [
+          {"and": [
+            "Gravity",
+            {"or": [
+              {"and": [
+                "Grapple",
+                {"simpleCycleFrames": 105},
+                {"cycleFrames": 20}
+              ]},    
+              {"and": [
+                {"or": [
+                  "Plasma",
+                  "Wave",
+                  "Spazer"
+                ]},
+                {"simpleCycleFrames": 115},
+                {"cycleFrames": 20}    
+              ]},
+              {"and": [
+                {"simpleCycleFrames": 115},
+                {"cycleFrames": 30}    
+              ]}
+            ]}
+          ]},
+          {"and": [
+            "Grapple",
+            {"simpleCycleFrames": 115},
+            {"cycleFrames": 20}
+          ]},
+          {"and": [
+            {"or": [
+              "Plasma",
+              "Wave",
+              "Spazer"
+            ]},
+            {"cycleFrames": 235}
+          ]},
+          {"cycleFrames": 275}
```

Comment:
This is a very scary farm.  So does cycleframe scaling mean you can be really safe?

Reply:
Yeah, there's no `simpleCycleFrames` here, only `cycleFrames`, so the full scaling would apply.